### PR TITLE
IP leak fix

### PR DESCRIPTION
**Pull request by yili1liu**
# Existing Sample Changes
## Description

Added some missing trademark symbols for FPGA found by an IP leak scan tool.

### CODE REVIEW

**Review by JoeOster — APPROVED** [review]

**Comment by JoeOster** [comment]
@yili1liu please fix your conflicting files

**Comment by yili1liu** [comment]
fixed conflicting files

**Review by JoeOster — DISMISSED** [review]

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp [hunk 1]
```diff
@@ -271,8 +271,8 @@ int main(int argc, char** argv) {
     // Instantiate multiple versions of the kernel
     // The II achieved by the compiler can differ between FPGA architectures
     //
-    // On Arria 10, we are able to achieve an II of 1 for all versions of the
-    // kernel.
+    // On Arria® 10, we are able to achieve an II of 1 for versions 1 and 2 of
+    // the kernel (not version 0).
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp:275]: This does not seem like a valid change.  I suspect you have accidentally reverted to an older version of this file?

**Review by mdbtucker — COMMENTED** [review]
Rest of the changes look good, just that one files seems to be messed up a bit.  Thanks for fixing these.

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp (hunk 1 above)
Comment (yili1liu) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp:275]: Yes, will fix that now

**Comment by mdbtucker** [comment]
Joe, are you still planning to merge this?  I haven't seen an update since my last review.

**Review by JoeOster — APPROVED** [review]

**Review by mdbtucker — APPROVED** [review]

**Pull request merged**